### PR TITLE
Fix method name (typo)

### PR DESCRIPTION
**Pull request by avoelkl**
### Description
This PR deprecates a current method with a typo and introduces the new method with correct spelling

### Fixed Issues (if relevant)
none

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@avoelkl thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: app/code/Magento/Multishipping/Block/Checkout/Overview.php [hunk 1]
```diff
@@ -319,9 +319,18 @@ public function getQuote()
     }
 
     /**
-     * @return mixed
```
Comment (dmanners) [inline app/code/Magento/Multishipping/Block/Checkout/Overview.php]: I think we might want to keep the return here

**Review by dmanners — CHANGES_REQUESTED** [review]
Just one small change

**Comment by avoelkl** [comment]
Hi @dmanners, I updated the PR and re-added the return in the doc.
Please note that I also reverted one commit that accidentally made it into this branch.

**Comment by dmanners** [comment]
Thanks @avoelkl I will process this now.

**Review by dmanners — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmanners, thank you for the review.
[ENGCOM-1675](https://jira.corp.magento.com/browse/ENGCOM-1675) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @avoelkl. Thank you for your contribution. 
Changes from your Pull Request will be available with the upcoming 2.2.6 release.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

File: app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
  */
 
 /**
- * Multishipping checkout base abstract block
+ * Mustishipping checkout base abstract block
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php:8]: Multishipping was correct

File: app/code/Magento/Multishipping/Block/Checkout/Shipping.php [hunk 3]
```diff
@@ -9,7 +9,7 @@
 use Magento\Quote\Model\Quote\Address;
 
 /**
- * Multishipping checkout shipping
+ * Mustishipping checkout shipping
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/Block/Checkout/Shipping.php:12]: Multishipping was correct

File: app/code/Magento/User/Model/Backend/Config/ObserverConfig.php [hunk 4]
```diff
@@ -72,7 +72,7 @@ public function getAdminPasswordLifetime()
     }
 
     /**
-     * Get admin maximum security failures from config
+     * Get admin maxiumum security failures from config
```
Comment (ihor-sviziev) [inline app/code/Magento/User/Model/Backend/Config/ObserverConfig.php:75]: Maximum was correct

**Pull request merged**